logic/transaction.js

```diff
@@ -964,15 +983,14 @@ Transaction.prototype.undo = function(transaction, block, sender, cb) {
  * unconfirmed balance negative amount.
  * Calls `applyUnconfirmed` based on transaction type (privateTypes). If error merge
  * account with amount.
+ *
  * @see privateTypes
```

Comment:
Is there a link we can provide for this?

Reply:
Yes, will be included in the next commits :)